src/assets/networks-config.yaml

```diff
@@ -12,11 +12,11 @@
 paths:
   vdeSwitch: /opt/rancher-desktop/bin/vde_switch
   vdeVMNet: /opt/rancher-desktop/bin/vde_vmnet
-  varRun: /private/var/run/rancher-desktop-lima
+  varRun: /private/var/run
```

Comment:
This is not enough; we don't update `_config/network.yaml` during an update (because the user may have customized the address ranges).

So we need some code to update `paths.varRun` when it is currently set to `/private/var/run/rancher-desktop-lima`.

Reply:
Actually, this _accidentally_ works…
https://github.com/rancher-sandbox/rancher-desktop/blob/242ecf1d058b084bae2442832419221d4bab59ec/src/k8s-engine/lima.ts#L1025-L1032

If `paths.varRun` isn't what we expect, we replace the file wholesale. And because we're changing `paths.varRun` itself, this will never match, so we'll replace the file…

Maybe we need to do something to manage migration better?

Reply:
Ok, it looks like the file gets changed, but it still asked for a password to create the old path (and did create it again, even though the actual files were created in `/var/run` using the new names:

```console
$ ls -ld /var/run/rancher*
drwxrwx---  3 daemon  everyone  96  7 Mar 16:53 /var/run/rancher-desktop-bridged_en0.ctl
dr-xr-xr-x  2 root    daemon    64  7 Mar 16:43 /var/run/rancher-desktop-lima
drwxrwx---  3 daemon  everyone  96  7 Mar 16:53 /var/run/rancher-desktop-shared.ctl
```

Not sure why there are 10min between creating the directory and the files, but maybe I just let the app sit on the sudo password prompt. I'm pretty sure the "explainer" dialog still mentioned creating the old path as well.

Reply:
Ok, it looks like the file gets changed, but it still asked for a password to create the old path (and did create it again, even though the actual files were created in `/var/run` using the new names:

```console
$ ls -ld /var/run/rancher*
drwxrwx---  3 daemon  everyone  96  7 Mar 16:53 /var/run/rancher-desktop-bridged_en0.ctl
dr-xr-xr-x  2 root    daemon    64  7 Mar 16:43 /var/run/rancher-desktop-lima
drwxrwx---  3 daemon  everyone  96  7 Mar 16:53 /var/run/rancher-desktop-shared.ctl
```

Not sure why there are 10min between creating the directory and the files, but maybe I just let the app sit on the sudo password prompt. I'm pretty sure the "explainer" dialog still mentioned creating the old path as well.

Reply:
NIce catch! Fixed now to not create that directory. We _do_ create `/private/var/run` if that's missing, but that shouldn't happen.